spec/supported_versions.rb

```diff
@@ -0,0 +1,3 @@
+def supported_versions
```

Comment:
can you move the file?
https://github.com/voxpupuli/puppet-zabbix/blob/master/spec/spec_helper_acceptance.rb#L6
we've some autoloader logic here. If you do that, you don't need to add the require to all spec files.

Reply:
First I added in `/spec/support/acceptance` but I'm using it in some unit tests as well.
Any idea how I could tackle that?

Reply:
indeed! https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb.erb#L30
our spec_helper.rb can be extended with additional lines. You can see an example here:
* https://github.com/voxpupuli/puppet-collectd/blob/master/spec/spec_helper.rb#L20
* https://github.com/voxpupuli/puppet-collectd/commit/c5e7f6d1d00c7bce637b31118873773a4a9aa9ff#diff-b1bbc4d50c1c098ca18224cbc9519ad646dcc5e3dd912edf55610ab5bba3566eL12-L13

So I suggest:
* move the file into the acceptance dir
* update .sync.yml
* update spec_helper.rb

Reply:
Just to be sure, from .sync.yml I need to remove the `spec/spec_helper.rb` line? (along with mock_with :stuck_out_tongue: )

Reply:
My bad, i need the spec_overrides property.